examples/generative/corrdiff/train.py

```diff
@@ -233,6 +237,9 @@ def main(cfg: DictConfig) -> None:
     elif precond == "resloss":
         c.network_kwargs.class_name = "modulus.models.diffusion.EDMPrecondSR"
         c.loss_kwargs.class_name = "modulus.metrics.diffusion.ResLoss"
+    elif precond == "reslossv2":
```

Comment:
Can we use a more descriptive name instead of `reslossv2` and 'EDMPrecondSRV2'? Also, which preconditioner is  recommended to be used?

Reply:
There isn't a recommendation on which preconditioner to use at this point, it appears to vary based on model.

Reply:
I think we can leave it as is for now, thanks